modules/panorama/main.tf

```diff
@@ -70,7 +70,14 @@ resource "azurerm_virtual_machine" "panorama" {
   }
 
   os_profile_linux_config {
-    disable_password_authentication = false
+    disable_password_authentication = var.password == null ? true : false
```

Comment:
Same point as in this thread https://github.com/PaloAltoNetworks/terraform-azurerm-vmseries-modules/pull/191/files#r984585354

Reply:
fixed